recipe_scrapers/wikicookbook.py

```diff
@@ -38,3 +38,25 @@ def instructions(self):
         return "\n".join(
             [normalize_string(instruction.get_text()) for instruction in instructions]
         )
+
+    def description(self):
+        paragraphs = list()
+        for tag in self.soup.find(class_="mw-parser-output"):
+            try:
+                # get all paragraphs except for links
+                if (
+                    tag.text.strip()
+                    and tag.name == "p"
+                    and not tag.find("span", {"id": "displaytitle"})
```

Comment:
See above.

Reply:
If `find()` didn't return an iterable object, this code would never run as it would raise a `TypeError` before entering the loop :)